post/2023/beyond-sf/index.qmd

```diff
@@ -0,0 +1,301 @@
+---
+title: "R-spatial beyond `{sf}`"
+author: 
+  - name: Josiah Parry
+    affiliation: Esri
+    orcid: 0000-0001-9910-865X
+categories: 
+  - posts
+  - rstats
+tags: [rspatial]
+date: "2023-09-06"
+slug: r-spatial-beyond-sf
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+**sf** has transformed the R ecosystem and helped pave the way for wider adoption of geospatial workflows by R users. The R-spatial ecosystem, however, is much bigger than just sf. There are lower level packages such as **geos**, **wk**, and **s2** that provide access to geometries and algorithms outside of the context of sf. These packages are often more performant than sf but require a different frame of mind when using them.
```

Comment:
It would be nice to have one plot comparing the performance of {geos}, {rsgeo} and {sf} in different operations. Each panel in the plot would present a different operation. I think it could encourage users.

Reply:
And maybe it is also worth quoting Edzer's publication or book?

Reply:
Perhaps there can be a follow up that does a bench marking of various operations? I think the one bench mark is sufficient for a small motivation. I don't want a bench mark to detract from the goal of "there are other types of geometry vectors you can use"

Reply:
+1 to follow-up. Lots of small blogs better than one huge one I reckon.

Reply:
I mean to give general reference only. But I think [section 7.1](https://r-spatial.org/book/07-Introsf.html#sec-sfintro) complements your text well.